lib/Widget/evaluationForms/post-hos/post-hos-day0/recovery_readiness_form.dart

```diff
@@ -0,0 +1,376 @@
+                                              _dizzy = value;
+                                              _readyRecover =
+                                                  completeAllCheckBox();
+                                            });
+                                          },
+                                          title: Text(
+                                              '7. ไม่มีหน้ามืดหรือเวียนศีรษะ'),
+                                        ),
+                                        CheckboxListTile(
+                                          value: _vomit,
+                                          selected: _vomit,
+                                          controlAffinity:
+                                              ListTileControlAffinity.leading,
+                                          activeColor: Color(0xFFC37447),
+                                          onChanged: (value) {
+                                            setState(() {
+                                              _vomit = value;
+                                              _readyRecover =
+                                                  completeAllCheckBox();
+                                            });
+                                          },
+                                          title: Text(
+                                              '8. ไม่มีอาการคลื่นไส้ อาเจียน'),
+                                        ),
+                                      ],
+                                    ),
+                                  ),
+                                  Padding(
+                                    padding: const EdgeInsets.all(8.0),
+                                    child: Container(
+                                      child: RaisedButton(
+                                          shape: RoundedRectangleBorder(
+                                              borderRadius:
+                                                  BorderRadius.circular(7.0)),
+                                          textColor: Colors.white,
+                                          padding: EdgeInsets.all(15),
+                                          color: Color(0xFF2ED47A),
+                                          child: Text('ยืนยัน',
+                                              style: TextStyle(fontSize: 18)),
+                                          onPressed: () async {
+                                            if (_knowFullName |
+                                                    _knowTime |
+                                                    _knowPlace |
+                                                    _fever |
+                                                    _breathGasp |
+                                                    _palpitations |
+                                                    _dizzy |
+                                                    _vomit !=
```

Comment:
Is this mean all must be false or only vomit is false?

By the way why not use false.

Reply:
true is means that field is checked.
This condition will just check whether the user has checked(ติ๊กที่ช่อง) an item or not when pressed the submit button.
However, I will change the name to make it more understandable.

Reply:
<img width="544" height = "300" alt="Screen Shot 2564-04-26 at 20 48 18" src="https://user-images.githubusercontent.com/57941667/116093271-bbda8200-a6d0-11eb-94cd-9bda58c68450.png">